common/logger/src/libra_logger.rs

```diff
@@ -125,10 +127,24 @@ impl LibraLoggerBuilder {
     }
 
     pub fn init(&mut self) {
-        self.build()
+        self.build();
     }
 
-    pub fn build(&mut self) {
+    pub fn build(&mut self) -> Arc<LibraLogger> {
+        let filter = {
+            let mut filter_builder = Filter::builder();
+
+            if env::var(LIBRA_LOG).is_ok() {
+                filter_builder.with_env(LIBRA_LOG);
+            } else if env::var(RUST_LOG).is_ok() {
+                filter_builder.with_env(RUST_LOG);
+            } else {
+                filter_builder.filter_level(self.level.into());
+            }
```

Comment:
Not in this case. The level in the builder is always "set" since its not optional. The environment variables are use to override the default behavior.